app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -68,87 +68,62 @@ class GithubWebhook extends RequestHandler<Body> {
   }
 
   Future<void> _handlePullRequest(String rawRequest) async {
-    final PullRequestEvent event = await _getPullRequest(rawRequest);
-    final List<IssueLabel> existingLabels =
-        _getProperty<List<dynamic>>(rawRequest, 'labels')
-            .cast<Map<String, dynamic>>()
-            .map<IssueLabel>(IssueLabel.fromJSON)
-            .toList();
-    final bool isDraft = _getProperty(rawRequest, 'draft');
+    final PullRequest event = await _getPullRequest(rawRequest);
```

Comment:
Is a pull request the only event we can get here? if not how do we optimize to only process pull requests?

Reply:
In that case I'd recommend renaming the variable to something like pullRequest. It will be easier to correlate to the object.

Reply:
Renamed it to "pr".